02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>this should probably be the first property mentioned, as knowing which levels you have helps to understand and explain the parent-child relationship</p><p>Move this to before the dimension property</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>